k8s/runners/staging/release.yaml

```diff
@@ -67,4 +68,4 @@ data:
               "gitlab-ci/spack_spec_needs_rebuild" = "$SPACK_SPEC_NEEDS_REBUILD"
             [runners.kubernetes.node_selector]
               "kubernetes.io/arch" = "amd64"
-              "spack.io/node-pool" = "glr-xlarge-pub"
+              "spack.io/node-pool" = "glr-large-pub"
```

Comment:
Let's have the testing and staging runners run on the same `glr-large-testing-pub` pool.  Be sure to set the pool's capacity to the sum of the two runners' `concurrency` settings + 8 instances of margin.

Reply:
Ah, I had this locally but didn't add it.  I think we have, 5 from testing + 3 from staging + 8 buffer = 16 which is the MaxSie right now.

Reply:
Perfect!